tests/test_bugzilla.py

```diff
@@ -46,11 +46,26 @@ def bughandler(bug, data):
             data[bug['id']] = bug
 
         bugs = {}
+
+        # Query string
         bugzilla.Bugzilla('bug_id=12345%2C12346&bug_id_type=anyexact&list_id=12958345&resolution=FIXED&query_format=advanced', bughandler=bughandler, bugdata=bugs).get_data().wait()
 
         self.assertEqual(bugs[12345]['id'], 12345)
         self.assertEqual(bugs[12346]['id'], 12346)
 
+        # Dict query
```

Comment:
Can you reinitialize the `bugs` variable here (otherwise the test could pass even if the class does nothing) or move the following to another function in the same class (e.g. `test_search_dict`)?